### PR TITLE
import rewards components

### PR DESCRIPTION
**Pull request by wellingtonfds**

### CODE REVIEW

File: src/components/reward/RewardsList.tsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 import styled from 'styled-components';
 import tx from 'tailwind-styled-components';
-import { CardReward } from './CardReward';
+import { RewardPools } from './reward-pools/RewardPools';
+
 
 export const RewardsList = () => {
   return (
-    <CardReward banner={banner.src} collectionImage={collection.src} />
+    <RewardPools chainId={5} />
```
Comment (paulobarcelosbr) [inline src/components/reward/RewardsList.tsx:9]: should this parameter be fixed?

File: src/components/shared/design-system/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import { chainConfig } from '@config/chain'
```
Comment (paulobarcelosbr) [inline src/components/shared/design-system/Button.tsx:1]: should you be using this legacy component?

File: src/components/reward/RewardsList.tsx (hunk 1 above)
Comment (wellingtonfds) [reply src/components/reward/RewardsList.tsx:9]: for that, because I will check this rigth now

**Review by paulobarcelosbr — APPROVED** [review]

File: .env.development [hunk 3]
```diff
@@ -9,7 +9,7 @@
 NEXT_PUBLIC_CHAIN_ID=5
 NEXT_PUBLIC_RESERVOIR_API_BASE=https://api-goerli.reservoir.tools
 NEXT_PUBLIC_PROXY_API_BASE=/api/reservoir
-NEXT_PUBLIC_NFTFY_API_BASE=https://us-east1-nftfy-backend-develop.cloudfunctions.net/v2
+NEXT_PUBLIC_NFTFY_API_BASE=https://us-east1-nftfy-backend-develop.cloudfunctions.net/v2/graphql
```
Comment (fabriciomirandabr) [inline .env.development:12]: Recomendo usar a base e só extender o parametro graphql na config

File: src/ChainConfig.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    scanAddress: 'https://goerli.etherscan.io/',
+    subgraphUrl: 'https://api.thegraph.com/subgraphs/id/Qmb2WL3s4gviNc6mvGzqUHHho1JuzicgvJnACA5rzb911k',
+    hub: {
+      contract: {
+        erc721Factory: '0x6b93fd2A7174FAA28265dC1d85ee603c3d7C36e8'
+      }
+    }
+  },
+  {
+    chainId: 80001,
```
Comment (fabriciomirandabr) [inline src/ChainConfig.ts:36]: Não damos suporte a polygon

File: src/components/reward/RewardsList.tsx (hunk 1 above)
Comment (fabriciomirandabr) [reply src/components/reward/RewardsList.tsx:9]: this parameter needs to be dynamic

**Pull request merged**